app/views/assignments/_read.html.erb

```diff
@@ -59,30 +59,30 @@
         <% if @penalty.type == "GracePeriodSubmissionRule" %>
           <% if !@grouping.nil? %>
               <%= raw(t('student.group_credits',
-                          available_grace_credits: 
+                          available_grace_credits:
                           @grouping.available_grace_credits)) %>
           <% end %>
         <% elsif @penalty.type == "PenaltyDecayPeriodSubmissionRule" %>
-          <ul>
+          <ul><br>
```

Comment:
It's a little unusual to put the `<br>` within the `<ul>`. Does it not work to put it before the `<ul>` tag?